moped-database/migrations/1692285495049_delete_work_type_construction_inspection/up.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM moped_work_types WHERE KEY = 'construction_inspection';
```

Comment:
typically we would not hard delete a work type: there is a `is_deleted` column. but this feature has not reached production yet 😄